### PR TITLE
ci: Change from setup-gradle action to ./gradlew command usage

### LINKED ISSUE #16704: Manually update the "gradle/actions/setup-gradle" action to v4.2.1
The action needs to be manually updated and then the steps must be changed according to this depreciation - https://github.com/gradle/actions/blob/main/docs/deprecation-upgrade-guide.md#using-the-action-to-execute-gradle-via-the-arguments-parameter-is-deprecated

### PR DESCRIPTION
**Pull request by mishomihov00**
**Description**:

Change from setup-gradle action to ./gradlew command usage. This is needed due to this change in v4.0.0 of the setup-gradle action - https://github.com/gradle/actions/blob/v4.0.0-rc.1/docs/deprecation-upgrade-guide.md#using-the-action-to-execute-gradle-via-the-arguments-parameter-is-deprecated

**Related issue(s)**:

Fixes #16704 

**Notes for reviewer**:

Test `.github/workflows/node-zxf-snyk-monitor.yaml` workflow run: https://github.com/hashgraph/hedera-services/actions/runs/12011679683

Test `.github/workflows/node-zxc-build-release-artifact.yaml` workflow run: https://github.com/hashgraph/hedera-services/actions/runs/12025886007 (This URL is new, after the gradle-build action change to ./gradlew)

All other workflows are tested within the checks of this PR.

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: .github/workflows/node-zxc-build-release-artifact.yaml [hunk 1]
```diff
@@ -828,14 +810,11 @@ jobs:
...
         env:
           NEXUS_USERNAME: ${{ secrets.sdk-ossrh-username }}
           NEXUS_PASSWORD: ${{ secrets.sdk-ossrh-password }}
-        with:
-          gradle-version: ${{ inputs.gradle-version }}
-          arguments: "release${{ inputs.release-profile }} -PpublishingPackageGroup=com.swirlds -Ps01SonatypeHost=true -PpublishSigningEnabled=true --scan --no-configuration-cache"
+        run: ./gradlew release${{ inputs.release-profile }} -PpublishingPackageGroup=com.swirlds -Ps01SonatypeHost=true -PpublishSigningEnabled=true --scan --no-configuration-cache
 
       - name: Gradle Publish Services to ${{ inputs.version-policy == 'specified' && 'Maven Central' || 'Google Artifact Registry' }} (${{ inputs.release-profile }})
         uses: gradle/gradle-build-action@ac2d340dc04d9e1113182899e983b5400c17cda1 # v3.5.0
```
Comment (rbarkerSL) [inline .github/workflows/node-zxc-build-release-artifact.yaml:820]: This should also be updated to use the `run: ./gradlew` syntax. `gradle/gradle-build-action` has been superseded by `setup-gradle`

**Review by rbarkerSL — CHANGES_REQUESTED** [review]

**Review by andrewb1269hg — DISMISSED** [review]
This looks good to me but I'd like @jjohannes or @rbarkerSL to confirm

File: .github/workflows/node-zxc-build-release-artifact.yaml [hunk 2]
```diff
@@ -828,24 +810,18 @@ jobs:
...
       - name: Gradle Publish Services to ${{ inputs.version-policy == 'specified' && 'Maven Central' || 'Google Artifact Registry' }} (${{ inputs.release-profile }})
-        uses: gradle/gradle-build-action@ac2d340dc04d9e1113182899e983b5400c17cda1 # v3.5.0
         if: ${{ inputs.dry-run-enabled != true && inputs.release-profile != 'none' && !cancelled() && !failure() }}
         env:
           NEXUS_USERNAME: ${{ secrets.svcs-ossrh-username }}
           NEXUS_PASSWORD: ${{ secrets.svcs-ossrh-password }}
-        with:
-          gradle-version: ${{ inputs.gradle-version }}
-          arguments: "release${{ inputs.release-profile }} -PpublishingPackageGroup=com.hedera.hashgraph -PpublishSigningEnabled=true --scan --no-configuration-cache"
+        run: release${{ inputs.release-profile }} -PpublishingPackageGroup=com.hedera.hashgraph -PpublishSigningEnabled=true --scan --no-configuration-cache
```
Comment (rbarkerSL) [inline .github/workflows/node-zxc-build-release-artifact.yaml:824]:
```suggestion
        run: ./gradlew release${{ inputs.release-profile }} -PpublishingPackageGroup=com.hedera.hashgraph -PpublishSigningEnabled=true --scan --no-configuration-cache
```

**Review by rbarkerSL — CHANGES_REQUESTED** [review]
One minor fix and then this should be good to go

**Review by rbarkerSL — APPROVED** [review]

**Pull request merged**